dbt_platform_helper/domain/pipelines.py

```diff
@@ -32,22 +34,29 @@ def __init__(
         self.abort = abort
         self.get_git_remote = get_git_remote
         self.get_codestar_arn = get_codestar_arn
+        self.terraform_manifest_provider = terraform_manifest_provider
+        self.ecr_provider = ecr_provider
 
-    def generate(self, terraform_platform_modules_version, deploy_branch):
-        pipeline_config = self.config_provider.load_and_validate_platform_config()
+    def generate(self, cli_terraform_platform_modules_version, deploy_branch):
```

Comment:
Should we add type hints for the function?

Reply:
I think this suggestion applies in other places too.

Reply:
@WillGibson I think I have them all now.